tests/test-spi.js

```diff
@@ -70,10 +70,14 @@ for (var i = 0; i < spiBusBits.length; i++) {
     spiBus = spi.open(init);
 
     for (var j = 0; j < dataNumber.length; j++) {
-        var bufferNumber = spiBus.transceive(deviceNum, dataNumber[j]);
-        assert(bufferNumber.toString("hex") === checkNumber[i][j],
-               "SPIBusTransceive: read and write number array by bits '" +
-               spiBusBits[i] + "'");
+        if (spiBusBits[i] === 16 && j === 1) {
```

Comment:
Can you add comments here like why we are removing this specific test case for 16bit mode?  lgtm otherwise.

Reply:
If I understand correctly it is in reference to issue #1445, its removing the invalid test case.